src/firmware_new/boards/frankie_v1/main.c

```diff
@@ -204,11 +264,32 @@ int main(void)
     /* USER CODE END RTOS_QUEUES */
 
     /* Create the thread(s) */
-    /* creation of defaultTask */
-    defaultTaskHandle = osThreadNew(StartDefaultTask, NULL, &defaultTask_attributes);
+    /* creation of NetStartTask */
+    NetStartTaskHandle = osThreadNew(io_proto_multicast_startNetworkingTask, NULL,
+                                     &NetStartTask_attributes);
+
+    /* creation of RobotStatusTask */
+    RobotStatusTaskHandle =
+        osThreadNew(io_proto_multicast_sender_Task,
+                    (void *)tbots_robot_msg_sender_profile, &RobotStatusTask_attributes);
+
+    /* creation of VisionMsgTask */
+    VisionMsgTaskHandle =
+        osThreadNew(io_proto_multicast_listener_Task, (void *)vision_msg_listener_profile,
+                    &VisionMsgTask_attributes);
+
+    /* creation of PrimMsgTask */
+    PrimMsgTaskHandle =
+        osThreadNew(io_proto_multicast_listener_Task,
+                    (void *)primitive_msg_listener_profile, &PrimMsgTask_attributes);
+
+    /* creation of testMsgUpdate */
+    testMsgUpdateHandle =
+        osThreadNew(test_msg_update, (void *)tbots_robot_msg_sender_profile,
+                    &testMsgUpdate_attributes);
```

Comment:
wondering if we want to have different targets for testing

Reply:
By "target", do you mean "firmware binary"?

Reply:
that's right, I meant it in the sense of running a firmware target to flash the robot

Reply:
Ah, IMO that's a bit overkill. If anything maybe a hardware switch on the robot to prevent test messages affecting robots during games?

Reply:
this will get nuked when we actually have a robot status updating task